<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>940</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1200</width>
    <height>940</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Modeling Program</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow {
  	background-color: #222831;  /* Black */
}

QGroupBox {
    background-color: #393E46;  /* Gray */
    color: #EEEEEE;
    border-radius: 8px;
    border: 3px solid #903749;
    margin-top: 6px;
    padding-top: 8px;
}

QGroupBox::title {
    subcontrol-origin: margin;
    subcontrol-position: top left;
    left: 10px;
	/* background-color: #393E46; 
	 border: 2px solid #00ADB5;*/
}

QLabel {
    color: #EEEEEE;
}

QPushButton {
	background-color: #820000;  
    color: #EEEEEE;
    border: 2px solid #903749;
    border-radius: 8px ;
    padding: 4px 8px;
    text-align: center;
    text-decoration: none;
    display: inline-block;
    font-size: 14px;
}

QPushButton::enabled { 
    background-color:  #2B2E4A;
 }

 QPushButton::hover {
	background-color: #03C988;  
 }

QCheckBox {
	color: #EEEEEE;
}

QCheckBox::indicator:checked {
    background-color:       red;
    border:                 2px solid white;
}

QCheckBox::indicator:unchecked {
    background-color:       black;
    border:                 2px solid white;
}

QRadioButton {
	color: #EEEEEE;
}

QRadioButton::indicator:checked {
    background-color:       red;
    border:                 2px solid white;
}

QRadioButton::indicator:unchecked {
    background-color:       black;
    border:                 2px solid white;
}


QDoubleSpinBox{
  	background-color: #222831; 
	color: #EEEEEE;
}
QSpinBox{
  	background-color: #222831; 
	color: #EEEEEE;
}

QListWidget {
     selection-color: #03C988;
	  selection-background-color: #222831; 
     padding-left: 5px;
     padding-right: 5px;
	  background-color: #222831; 
	  color: #EEEEEE;
}

QListWidget::item:selected {
  /*border-bottom: 1px solid #EEEEEE;*/
}
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QGraphicsView" name="GW_locator">
      <property name="minimumSize">
       <size>
        <width>860</width>
        <height>860</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>860</width>
        <height>860</height>
       </size>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QGroupBox" name="GB_main">
      <property name="title">
       <string/>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>20</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="HL_buttons">
         <property name="sizeConstraint">
          <enum>QLayout::SetMinimumSize</enum>
         </property>
         <item>
          <widget class="QPushButton" name="PB_modeling">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
             <horstretch>0</horstretch>
             <verstretch>50</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>40</height>
            </size>
           </property>
           <property name="text">
            <string>Смоделировать</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="PB_buildingGraphics">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
             <horstretch>0</horstretch>
             <verstretch>50</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>40</height>
            </size>
           </property>
           <property name="text">
            <string>Построить графики</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QGroupBox" name="GB_trajectory">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>100</height>
          </size>
         </property>
         <property name="title">
          <string>Траектория самолёта</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QRadioButton" name="RB_onLine">
            <property name="text">
             <string>Прямолинейная</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="RB_turn">
            <property name="text">
             <string>Поворот</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="RB_onCircle">
            <property name="text">
             <string>По окружности</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="GB_trajectoryParams">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>Параметры движения</string>
         </property>
         <layout class="QFormLayout" name="formLayout_2">
          <item row="0" column="0">
           <widget class="QLabel" name="L_velocity">
            <property name="text">
             <string>Скорость, м/с:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QDoubleSpinBox" name="DSB_velocity">
            <property name="minimum">
             <double>56.000000000000000</double>
            </property>
            <property name="maximum">
             <double>278.000000000000000</double>
            </property>
            <property name="singleStep">
             <double>1.000000000000000</double>
            </property>
            <property name="value">
             <double>138.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="L_sigmaRho">
            <property name="text">
             <string>σ шума p, м:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QDoubleSpinBox" name="DSB_sigmaRho">
            <property name="maximum">
             <double>50.000000000000000</double>
            </property>
            <property name="value">
             <double>15.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="L_sigmaTheta">
            <property name="text">
             <string>σ шума θ, ° :</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QDoubleSpinBox" name="DSB_sigmaTheta">
            <property name="maximum">
             <double>5.000000000000000</double>
            </property>
            <property name="singleStep">
             <double>0.100000000000000</double>
            </property>
            <property name="value">
             <double>0.500000000000000</double>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="L_updateTime">
            <property name="text">
             <string>Обновление данных, с:</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QDoubleSpinBox" name="DSB_updateTime">
            <property name="minimum">
             <double>1.000000000000000</double>
            </property>
            <property name="maximum">
             <double>15.000000000000000</double>
            </property>
            <property name="value">
             <double>5.000000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="GB_filters">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>Отображение траекторий</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QCheckBox" name="CB_withNoise">
            <property name="text">
             <string>Зашумленная</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="CB_ABfilter">
            <property name="text">
             <string>α-β фильтр</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="CB_ABfilterMNK">
            <property name="text">
             <string>α-β фильтр МНК</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="CB_KalmanFilterCV">
            <property name="text">
             <string>Фильтр Калмана CV</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="GB_filtersParams">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>Параметры фильтра</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QListWidget" name="LW_filters">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="currentRow">
             <number>0</number>
            </property>
            <item>
             <property name="text">
              <string>α-β фильтр</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>α-β фильтр МНК</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Фильтр Калмана CV</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <widget class="QStackedWidget" name="SW_filters">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>300</height>
             </size>
            </property>
            <property name="currentIndex">
             <number>0</number>
            </property>
            <widget class="QWidget" name="P_ABfilter">
             <layout class="QFormLayout" name="formLayout_3">
              <item row="0" column="0">
               <widget class="QLabel" name="LB_AB_Kmax">
                <property name="text">
                 <string>Предельное k: </string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QSpinBox" name="SB_AB_Kmax">
                <property name="minimum">
                 <number>5</number>
                </property>
                <property name="maximum">
                 <number>150</number>
                </property>
                <property name="value">
                 <number>25</number>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="P_ABfilterMNK">
             <layout class="QFormLayout" name="formLayout_4">
              <item row="0" column="0">
               <widget class="QLabel" name="LB_AB_MNK_Kmax">
                <property name="text">
                 <string>Предельное k:</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QSpinBox" name="SB_AB_MNK_Kmax">
                <property name="minimum">
                 <number>5</number>
                </property>
                <property name="maximum">
                 <number>150</number>
                </property>
                <property name="value">
                 <number>25</number>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="LB_AB_MNK_max">
                <property name="text">
                 <string>Кол-во значений МНК:</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QSpinBox" name="SB_AB_MNK_MNKmax">
                <property name="minimum">
                 <number>3</number>
                </property>
                <property name="maximum">
                 <number>150</number>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="P_KalmanFilterCV">
             <layout class="QFormLayout" name="formLayout_5">
              <item row="0" column="0">
               <widget class="QLabel" name="L_KalmanFilterCV_sigmaNoiseCoord">
                <property name="text">
                 <string>σ шума  коориданты, м:</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QDoubleSpinBox" name="DSB_KalmanFilterCV_sigmaNoiseCoord">
                <property name="minimum">
                 <double>0.010000000000000</double>
                </property>
                <property name="maximum">
                 <double>50.000000000000000</double>
                </property>
                <property name="singleStep">
                 <double>0.100000000000000</double>
                </property>
                <property name="value">
                 <double>10.000000000000000</double>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="L_KalmanFilterCV_sigmaNoiseVelocity">
                <property name="text">
                 <string>σ шума скорости , м/c:</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QDoubleSpinBox" name="DSB_KalmanFilterCV_sigmaNoiseVelocity">
                <property name="minimum">
                 <double>0.010000000000000</double>
                </property>
                <property name="maximum">
                 <double>50.000000000000000</double>
                </property>
                <property name="singleStep">
                 <double>0.100000000000000</double>
                </property>
                <property name="value">
                 <double>1.000000000000000</double>
                </property>
               </widget>
              </item>
              <item row="2" column="0">
               <widget class="QLabel" name="L_KalmanFilterCV_sigmaNoiseAcceleration">
                <property name="text">
                 <string>σ шума ускорения , м/c²:</string>
                </property>
               </widget>
              </item>
              <item row="2" column="1">
               <widget class="QDoubleSpinBox" name="DSB_KalmanFilterCV_sigmaNoiseAcceleration">
                <property name="minimum">
                 <double>0.000000000000000</double>
                </property>
                <property name="maximum">
                 <double>25.000000000000000</double>
                </property>
                <property name="singleStep">
                 <double>0.100000000000000</double>
                </property>
                <property name="value">
                 <double>0.000000000000000</double>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
